### ISSUE TITLE
Hive Index couldn't be created

### ISSUE THREAD

**Issue by uslss** [dbeaver/dbeaver#7530]
#### System information: 
- Operating system (distribution) and version win 10 x64
- DBeaver version 6.3.2

#### Connection specification:
- Database name and version Hadoop Hive

#### Describe the problem you're observing:
Index couldn't be created - an error appears
![image](https://user-images.githubusercontent.com/31996417/71782419-ad9fb880-2fea-11ea-9469-e04ac3f65ef8.png)

**Comment by uslss** [comment]
https://dbeaver.testquality.com defect [D52](https://dbeaver.testquality.com/project/8805/result/1761729/defect/6107). Do not remove or edit.

**Comment by serge-rider** [comment]
We don't support Hive indexes management.
They are very different than regular relational indexes.

Mark as FR.

**Comment by LonwoLonwo** [comment]
https://cwiki.apache.org/confluence/display/Hive/IndexDev#IndexDev-IndexingIsRemovedsince3.0 Indexing Is Removed since 3.0

**Comment by LonwoLonwo** [comment]
UNIQUE Indices not supported https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Create/Drop/AlterIndex

**Comment by LonwoLonwo** [comment]
So. Indexes are maintained in a separate table in Hive. Now they are marked with a different icon in the tree and cannot be removed from the tree. In DBeaver, we created entity indexes inside tables that contain references to index tables and can be dropped.
We also changed the dialog for creating indexes for Hive. We left the most important points. 
After creating the indexes, it is rebuilt.
The syntax for dropping indexes has also been changed.

![2020-11-12 00_46_47-DBeaver Enterprise 7 3 0 - _MySQL 8 0_ Script-53](https://user-images.githubusercontent.com/45152336/98872381-8118c080-2487-11eb-9232-d2dbd554fca1.png)
